### PR TITLE
fix(flow-execute): fixed log link for Windows

### PR DESCRIPTION
**Pull request by YuryShkoda**
## Intent

1. Fix log link for Windows.
2. Fix creating files on Windows.

## Implementation

1. Updated `flow execute` logic.
2. Updated `createFile` utility.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [x] Any new functionality has been unit tested.
- [x] All unit tests are passing (`npm test`).
Mocked:
image
Server:
image
- [x] All CI checks are green.
- [ ] JSDoc comments have been added or updated.
- [x] Reviewer is assigned.

### CODE REVIEW

File: src/utils/file.js [hunk 1]
```diff
@@ -183,6 +183,8 @@ export async function createFile(fileName, content, debug = false) {
   }
 
   return new Promise(async (resolve, reject) => {
+    fileName = fileName.split('/').join(path.sep)
```
Comment (krishna-acondy) [inline src/utils/file.js:186]: Definitely worth adding a test or two for this.

**Review by saadjutt01 — DISMISSED** [review]

File: src/utils/file.js (hunk 1 above)
Comment (YuryShkoda) [reply src/utils/file.js]: Moved this logic to the `unifyFilePath` utility and added unit test for it.

File: src/utils/spec/file.spec.ts [hunk 2]
```diff
@@ -41,4 +42,19 @@ describe('file utility', () => {
       deleteFolder(path.join(process.cwd(), 'testFolder_1'))
     })
   })
+
+  describe('unifyFilePath', () => {
+    it('should unify file path in Unix-like systems', () => {
```
Comment (krishna-acondy) [inline src/utils/spec/file.spec.ts:47]: Do you think it's worth adding a test case for when the separators are mixed up?

Comment (YuryShkoda) [reply src/utils/spec/file.spec.ts:47]: good point!
updated `unifyFilePath` utility

**Review by krishna-acondy — APPROVED** [review]

**Pull request merged**